packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -413,6 +413,8 @@
 	"nodeSettings.waitBetweenTries.displayName": "Wait Between Tries (ms)",
 	"nodeSettings.useTheHttpRequestNode": "Use the <b>HTTP Request</b> node to make a custom API call. We'll take care of the {nodeTypeDisplayName} auth for you. <a target=\"_blank\" href=\"https://docs.n8n.io/integrations/custom-operations/\">Learn more</a>",
 	"nodeSettings.scopes": "{count} scope available for {activeCredential} credentials<br>{scopes} | {count} scopes available for {activeCredential} credentials<br>{scopes}",
+	"nodeSettings.scopes.shortContent": "<a data-key=\"toggle-expand\">{count} scope</a> available for {activeCredential} credentials | <a data-key=\"toggle-expand\">{count} scopes</a> available for {activeCredential} credentials",
+	"nodeSettings.scopes.fullContent": "<a data-key=\"toggle-expand\">{count} scope</a> available for {activeCredential} credentials<br>{scopes}<br><a data-key=\"show-less\">Show less</a> | <a data-key=\"toggle-expand\">{count} scopes</a> available for {activeCredential} credentials<br>{scopes}<br><a data-key=\"show-less\">Show less</a>",
```

Comment:
could we come up with better names? maybe `scopes.notice` and `expandedNoticeWithScopes`